.env.development

```diff
@@ -9,7 +9,7 @@ SERVER_PORT=34343
 SECRET_KEY_BASE=FA11FA11FA11FA11FA11
 
 # Authentication setting
-AUTHENTICATION_SITE_URL='http://localhost:3000'
-AUTHENTICATION_REDIRECT_URI='http://localhost:4000/auth/defence_request/callback'
-AUTHENTICATION_APPLICATION_ID='xxx'
-AUTHENTICATION_APPLICATION_SECRET='xxx'
+AUTHENTICATION_SITE_URL='http://localhost:45454'
+AUTHENTICATION_REDIRECT_URI='http://localhost:34343/auth/defence_request/callback'
+AUTHENTICATION_APPLICATION_ID=8d142261e79fd579777c83bba6deffb9dea9499e696b41c839d101718819a48794d6b359bd51eae94b47c04fc1
```

Comment:
A few things:

1. Could we be consistent with quoting? Just makes things easier generally. You're welcome to remove all the quotes if you want.
2. If the application IDs contain things that are not actual secrets, could we try and set them to something that's 'obviously wrong'? I did this with 'SECRET_KEY_BASE' above. So you could set it to something appropriate from https://reminiscential.wordpress.com/2008/09/03/hexspeak-word-list/ if it has to be hex. Or you could just set it to 'not-actually-a-valid-auth-secret', for example. That just reduces the chances of someone copy-and-pasting things into production incorrectly because they don't truly understand that it needs to be set to something unique.
3. I didn't see a PR for the auth app that creates the matching secrets and application IDs? We need to make them match.
4. In the auth app, the ID value is referred to as a 'uid' - so I've set the app IDs to a generated uuid. Not sure if we should be consistent with them.